docs/getting-started.md

```diff
@@ -2,20 +2,267 @@
+  such as Injective and Sui. (If your chain doesn't have this concept, remove
+  this line).
+- `signMessage` is a message signing function obtained from the web3 API you use to
+  connect to the wallet, which takes a `Uint8Array` containing the message
+  and returns a `Promise<Uint8Array>` containing the signature.
+  Some wallet providers, such as Solana and EVM, can provide this function directly,
+  while others will require you to write an adapter.
+
+Here is a complete example for a card using Ethereum:
+
+```
+import { arrayify } from '@ethersproject/bytes';
+import {
+  NotifiContext,
+  NotifiSubscriptionCard,
+} from '@notifi-network/notifi-react-card';
+import '@notifi-network/notifi-react-card/dist/index.css';
+import { useEthers } from '@usedapp/core';
+import { providers } from 'ethers';
+import React, { useMemo } from 'react';
+
+export const NotifiCard: React.FC = () => {
+  const { account, library } = useEthers();
+  const signer = useMemo(() => {
+    if (library instanceof providers.JsonRpcProvider) {
+      return library.getSigner();
+    }
+    return undefined;
+  }, [library]);
+
+  if (account === undefined || signer === undefined) {
+    // account is required
+    return null;
+  }
+
+  return (
+    <NotifiContext
+      dappAddress="<YOUR OWN DAPP ADDRESS HERE>"
+      env="Production"
+      signMessage={async (message: Uint8Array) => {
+        const result = await signer.signMessage(message);
+        return arrayify(result);
+      }}
+      walletPublicKey={account}
+      walletBlockchain="ETHEREUM" // NOTE - Please update to the correct chain name.
+      //If Polygon, use "POLYGON"
+      //If Arbitrum, use "ARBITRUM"
+      //If Binance, use "BINANCE"
+      //If Optimism, use OPTIMISM
+    >
+      <NotifiSubscriptionCard
+        cardId="<YOUR OWN CARD ID HERE>"
+      />
+    </NotifiContext>
+  );
+};
+```
+
+(TODO: Screenshots for setting up the Notifi React card)
+
+Once you have this set up, run it and navigate to the card.
+Connect your wallet to the site and follow the wallet prompts
+in order to sign into the card.
+
+Once you've done that, you'll see prompts to enter your email
+address to sign up for notifications. Enter the email address you
+want to receive notifications on (anything you can verify will do),
+and then simply check off the Community Manager topic you created
+before! Notifi will send you a verification email - once you click
+the link to verify your email, you'll be all set to receive
+notifications!
+
+(TODO: Screenshots of using the Notifi React card)
+
+## Sending Your First Message
+
+Finally, now that we have a topic and a user signed up to it,
+it's time to send a message!
+
+1. Navigate to the **Community Manager** page in Admin Portal.
+2. Under **Audience/Notification Topic**, select the Community Manager
+   topic you've created.
+3. Enter some text in the **Campaign Name** field. Your audience won't
```

Comment:
Terms and jargon that we use should ideally link to some page that explains in much more depth. One example here would be linking this Campaign Name to a "Using Community Manager" or something like that section where it goes into depth

Reply:
Agreed. I've added most of the "check out more stuff" content at the bottom under the "Next Steps" section, but I also added a link here to a stub about our reporting features.